frontend/apps/picasso/components/Molecules/AllBondsTable.tsx

```diff
@@ -41,6 +47,27 @@ export const AllBondsTable: React.FC<AllBondsTableProps> = ({
   onRowClick = () => {},
   ...rest
 }) => {
+  const { loading, data, error } = useQuery(GET_BONDED_FINANCE);
```

Comment:
This is a small cleanup in the bond page, unrelated to the PR, can be removed. this page is disabled.